frontend/src/components/pages/profile/components/addPostForm/AddPostForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import Box from '@material-ui/core/Box';
+import SendSharpIcon from '@material-ui/icons/SendSharp';
+import './addPostForm.scss';
+
+const AddPostForm: React.FC = () => {
+  const [postValue, setPostValue] = useState<string>('');
+
+  const inputValueHandleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setPostValue(event.target.value);
+  };
+
+  const sendPost = (event: React.FormEvent) => {
+    event.preventDefault();
+    if (postValue === '' || postValue === ' ') {
+      alert('Please write something');
```

Comment:
it is not best approach for "required" validation

Reply:
Please syncup with @oleksandrstarov how to provide user-friendly validation in React and React Material ui